### PR TITLE
[C-875] Remove blocklist

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Removes blocklist feature from client, which is no longer needed now because of trusted-notifier.

### CODE REVIEW

File: packages/web/src/common/services/audius-backend/AudiusBackend.ts [hunk 1]
```diff
@@ -94,6 +94,7 @@ declare global {
     phantom: any
     solana: PhantomProvider
     web3: any
```
Comment (raymondjacobson) [inline packages/web/src/common/services/audius-backend/AudiusBackend.ts:96]: what even is lowercase web3? I feel like that doesn't exist actually

**Review by raymondjacobson — APPROVED** [review]
Thanks!

File: packages/web/src/common/services/audius-backend/AudiusBackend.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/common/services/audius-backend/AudiusBackend.ts:96]: yeah agreed, this looks like a bug

File: packages/web/src/components/search-bar/store/sagas.js [hunk 2]
```diff
@@ -35,16 +33,11 @@ export function* getSearchResults(searchText) {
 
   const { tracks, albums, playlists, users } = results
   const checkedUsers = users.filter((t) => !t.is_deactivated)
-  const checkedTracks = (yield call(
-    setTracksIsBlocked,
-    tracks,
-    audiusBackendInstance
-  )).filter((t) => !t.is_delete && !t._blocked && !t.user.is_deactivated)
```
Comment (sliptype) [inline packages/web/src/components/search-bar/store/sagas.js:42]: Do we still need to check is_delete and is_deactivated?

**Review by sliptype — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-875-remove-blist

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-875-remove-blist

**Pull request merged**